src/components/swap/SwapModalFooter.tsx

```diff
@@ -85,7 +84,7 @@ export default function SwapModalFooter({
   const label = `${trade.executionPrice.baseCurrency?.symbol} `
   const labelInverted = `${trade.executionPrice.quoteCurrency?.symbol}`
   const formattedPrice = formatNumber({
```

Comment:
will this overwrite the formatting you added in https://github.com/Uniswap/interface/pull/7301/files#diff-d0161b639d6c25d54a994e66cf2be7521eb5be41d7d8e7c723a07210b5fd9f5fR87 ?

Reply:
no it shouldn't - i didn't change anything from the base branch and thats the base branch

Reply:
oh sorry - yes it should because I am deleting this function. Sorry github was bugging out and couldn't see the blame